src/lib/tests/__tests__/MockRelayRendererFixtures.tsx

```diff
@@ -26,7 +26,8 @@ export const Artwork = createFragmentContainer(
     <View>
       <Image source={{ uri: props.artwork.image.url }} />
       <Metadata artworkMetadata={props.artwork} />
-      {props.artwork.artist && <ArtistQueryRenderer id={props.artwork.artist.gravityID} />}
+      {/* FIXME: Should this be a slug? */}
+      {props.artwork.artist && <ArtistQueryRenderer id={props.artwork.artist.slug} />}
```

Comment:
This ultimately is used with the `artist(id: ...)` MP resolver, so whichever works there 👍